exporter/datadogexporter/internal/logs/sender.go

```diff
@@ -64,30 +65,62 @@ func (s *Sender) SubmitLogs(ctx context.Context, payload []datadogV2.HTTPLogItem
 		s.logger.Debug("Submitting logs", zap.Any("payload", payload))
 	}
 
-	// enable sending gzip
-	// Get a fresh opts for each request to avoid duplicating ddtags
-	opts := *datadogV2.NewSubmitLogOptionalParameters().WithContentEncoding(datadogV2.CONTENTENCODING_GZIP)
+	var prevDdtags string
+	var logItemBatch []datadogV2.HTTPLogItem
```

Comment:
Can you create a group? And maybe make the variable names a bit more friendly as per our [coding guidelines](https://datadoghq.atlassian.net/wiki/spaces/OTEL/pages/2580119947/Go+Coding+Guidelines#Naming-things) ?

e.g. 
```suggestion
	var (
		prevtags string // ... 
		batch []datadogV2.HTTPLogItem // ... 
	)
```
Can you please document them too?